src/components/CartItem/CartItem.module.scss

```diff
@@ -0,0 +1,147 @@
+@import '@styles/variables';
+@import '@styles/breakpoints';
+
+.cart {
+  display: flex;
+  flex-direction: column;
+  width: 288px;
+  border: 1px solid $color-elements;
+  margin-bottom: 16px;
+  padding: 16px;
+  gap: 24px;
+  align-items: center;
+
+  @include bp-custom-up(640px) {
+    flex-direction: row;
+    width: 592px;
+    padding: 24px;
+  }
+
+  @include bp-x-large-up {
+    flex-direction: row;
+    width: 752px;
+  }
```

Comment:
fd-row should remain from breakpoint 640px but i`m not sure. try to check this out